Kernel/Syscall.cpp

```diff
@@ -291,6 +291,10 @@ static u32 handle(RegisterDump& regs, u32 function, u32 arg1, u32 arg2, u32 arg3
         IO::out16(0x604, 0x2000);
         break;
     }
+    case Syscall::SC_reboot: {
+        return current->process().sys$reboot();
+        break;
```

Comment:
No need for `break` after a `return` statements, as it's never reachable.

Reply:
Whoops... Missed that one...